projects/libiec61850/Dockerfile

```diff
@@ -20,3 +20,9 @@ RUN git clone https://github.com/mz-automation/libiec61850
 WORKDIR $SRC
```

Comment:
I presume we don't want to merge this and the `build.sh`?

Reply:
yep, removed.

Reply:
Ah this is outdated.